app/controllers/DigitalSubscription.scala

```diff
@@ -72,16 +73,32 @@ class DigitalSubscription(
 
   def digitalGeoRedirect: Action[AnyContent] = geoRedirect("subscribe/digital")
 
+  private def userHasDigipack(user: AuthenticatedIdUser): Future[Boolean] = {
+    user.credentials match {
+      case cookies: AccessCredentials.Cookies =>
+        membersDataService.userAttributes(cookies).value map {
+          case Left(_) => false
+          case Right(response: MembersDataService.UserAttributes) => response.contentAccess.digitalPack
+        }
+      case _ => Future.successful(false)
+    }
+  }
+
   def displayForm(): Action[AnyContent] =
     authenticatedAction(subscriptionsClientId).async { implicit request =>
       implicit val settings: AllSettings = settingsProvider.getAllSettings()
       identityService.getUser(request.user).fold(
         error => {
           SafeLogger.error(scrub"Failed to display digital subscriptions form for ${request.user.id} due to error from identityService: $error")
-          InternalServerError
+          Future.successful(InternalServerError)
         },
-        user => Ok(digitalSubscriptionFormHtml(user))
-      ).map(_.withSettingsSurrogateKey)
+        user => {
+          userHasDigipack(request.user) map {
+            case true =>  Redirect(routes.DigitalSubscription.displayThankYouExisting().url, request.queryString, status = FOUND)
```

Comment:
Minor: I would probably favour if/else here as I think we're just dealing with a Boolean